app/helpers/date_validation_helper.rb

```diff
@@ -35,6 +35,13 @@ def start_date_before_end_date
     end
   end
 
+  def start_date_before_current_year_and_month
+    if start_date.year > Time.zone.today.year || \
```

Comment:
I think you could use date comparison like `start_date  < Date.today` or is there a reason you decided to go with this solution?

Reply:
Only reason was that `end_date_before_current_year_and_month` was implemented this way so I followed suit. Should have questioned it, thank you for doing so!

Will refactor to your solution. 🚢

Reply:
Resolved in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1563/commits/9c97aff4905fa20157502cff588d0c9bed3822a7.

Reply:
Oh I think it was because of the start date does not have a day (or at least it is not displayed) and I assume it defaults to the first day of the month.  You could also use `Time.zone.today.at_beginning_of_month`, but I think it wouldn't make a difference.